applications/machine_learning/configuration/nrf5340dk_nrf5340_cpuapp/linker_xip.ld

```diff
@@ -0,0 +1,512 @@
+/*
+ * Copyright (c) 2013-2014 Wind River Systems, Inc.
```

Comment:
this file will be extremally difficult to maintain. I don't think this is the right approach.

What we could do:
* add cmake target to rename sections inside the library so these are not included by default
* add a change to linker script in zephyr so it would add exclusions based on the content of the provided list

Both approaches could actually be useful as these will be reusable by others. The approach with Zephyr should be easy to implement technically but harder to get it merged (maybe we can do it in our internal fork? @carlescufi @gmarull )